### ISSUE TITLE
Add a print button

### ISSUE THREAD

**Issue by claudiodsf** [IPGP/scientific_python_cheat_sheet#6]
Does the css theme have a printable version?

**Comment by tbelahi** [comment]
I don't think so. When we use tools to convert the HTML + CSS to pdf, it does not look good, maybe we did not find the right tool though. It is possible to make a nice screenshot of the webpage with firefox, as explained here: http://korben.info/faire-une-capture-ecran-complete-de-site-web-directement-depuis-firefox.html

**Comment by tbelahi** [comment]
Also if you do not care to much about the styling and want to print, you can use pandoc and the commands listed in the file conversion.sh

**Comment by MMesch** [comment]
To summarize. In firefox you can do:

```
shift + f2
screenshot out.png --fullpage
```

**Comment by MMesch** [comment]
ultimately we should have:
- the modules/subjects arranged in rows not columns.
- a print button that prints the page with a good layout. Doesn't cut stuff at pagebreaks

**Comment by QuLogic** [comment]
Regular Print Preview seems okay; the modules are listed in rows as you'd expect. The ToC takes up a lot of space and maybe could be hidden, though.
